modules/cnf-about-topology-aware-lifecycle-operator-config.adoc

```diff
@@ -0,0 +1,121 @@
+* Applicable list of managed policies
+* Number of concurrent updates
+* Applicable canary updates
+* Desired {product-title} version
+* Desired Operator version
+
+The `ClusterGroupUpgrade` CR can be in the following states:
+
+`UpgradeNotStarted`:: In this state, the `ClusterGroupUpgrade` CR is created and the `remediationAction` field is set to `inform`.
++
+The Operator builds a remediation plan based on the following fields:
++
+* The `clusterSelector` field specifies the labels applied to the clusters that you want to update
+* The `clusters` field specifies the list of clusters to update
+* The `canaries` fields specifies the clusters for canary updates 
+* The `maxConcurrency` field specifies the number of clusters to update in a batch
++
+The Operator transitions to the `UpgradeNotCompleted` state after the `remediationAction` field is set to `enforce` and the Operator starts to update the noncompliant clusters.
++
+[NOTE]
+====
+The administrator can only make changes to the fields that define the remediation plan in the `UpgradeNotStarted` state. 
+====
++
+.Sample CR
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu
+  namespace: default
+spec:
+  clusters: <1>
+  - spoke1
+  - spoke2
+  - spoke3
+  enable: false
+  managedPolicies:
+  - policy1-common-cluster-version-policy
+  - policy2-common-pao-sub-policy
+  - policy3-common-ptp-sub-policy
+  remediationStrategy: <2>
+    canaries: <3>
+    - spoke1
+    maxConcurrency: 2 <4>
+    timeout: 240
+status: <5>
+  conditions:
+  - message: The ClusterGroupUpgrade CR is not enabled
+    reason: UpgradeNotStarted
+    status: "False"
+    type: Ready
+----
+<1> The list of clusters to update.
+<2> The `remediationStrategy` fields define the specifics of the cluster updates.
+<3> The remediation plan starts with the clusters listed in the `canaries` field.
+<4> The `maxConcurrency` field defines the maximum number of concurrent updates in a batch. The number of remediation batches is the number of clusters divided by `maxConcurrency`.
+<5> The `status` field displays information about the status of the updates.
+
+`UpgradeNotCompleted`:: In this state, the Operator enforces the policies following the remediation plan defined in the `UpgradeNotStarted` state.
++
+Enforcing the policies of subsequent batches starts immediately after the current batch policies are all compliant. If the batch times out, the Operator moves on to the next batch. The value of this batch timeout is the `timeout` field divided by the number of batches in the remediation plan.
++
+The platform update is enforced first within a batch. After the platform update policy is compliant, the operator update policies are enforced.
++
+The Operator transitions to the `UpgradeTimedOut` state in two cases:
++
+* If the first batch of `ClusterGroupUpgrade` is set to `canaries` and the policies for this first batch are not compliant within the batch timeout
```

Comment:
```suggestion
* If the current batch is of canary type and the cluster in the batch doesn't become Compliant with all the managed policies within the batch timeout
```
There can be multiple canary clusters and in such a case each canary cluster will be in its own batch.

Reply:
Sounds like an important detail, I added a clarifying sentence to line 76:
> Each canary cluster forms a single-cluster batch.